lib/wcc/contentful/redirect.rb

```diff
@@ -1,33 +0,0 @@
-# frozen_string_literal: true
-require 'contentful_model'
-
-module WCC::Contentful
-  class Redirect < ContentfulModel::Base
```

Comment:
where'd this go?

Reply:
Deleted - I need to bump the version so The Porch doesn't pull in this code.  I figured that was OK since Jeremy was saying he can't use ContentfulModel with Preview anyways.